src/richie-front/js/components/CourseGlimpse/CourseGlimpse.spec.tsx

```diff
@@ -18,12 +19,14 @@ describe('components/CourseGlimpse', () => {
       name: 'Some Organization',
     } as Organization;
     const wrapper = render(
-      <CourseGlimpse course={course} organizationMain={organization} />,
+      <IntlProvider>
+        <CourseGlimpse course={course} organizationMain={organization} />
+      </IntlProvider>,
     );
 
     expect(wrapper.html()).toContain('Course 42');
     expect(wrapper.find('img').attr('src')).toContain('/thumbs/small.png');
-    expect(wrapper.html()).toContain('Starts on Mar 12, 2018');
+    expect(wrapper.html()).toContain('Starts on', 'Mar 12, 2018');
```

Comment:
Curious: why this change?

Reply:
There is now a `<span></span>` around the date

Reply:
oh then maybe it's a good idea to include the span in the test?